syntax = "proto3";

package Packet;

message GameBoardPacket {
	uint32 game_board_width = 1;
	uint32 game_board_height = 2;
	repeated string room_names = 3;
	repeated uint32 tiles = 4;
	repeated bool collision_map = 5;
}